src/algorithms.rs

```diff
@@ -80,6 +83,11 @@ impl Algorithm {
             Algorithm::EdDSA => AlgorithmFamily::Ed,
         }
     }
+
+    /// Converting Key Algorithm to Algorithm
+    pub fn from_key_algorithm(s: &KeyAlgorithm) -> Result<Self> {
```

Comment:
I think it makes more sense to do `KeyAlgorithm.to_algorithm` since this is not really meant to be public in practice and this way all the logic is contained in jwk.rs?

Reply:
Agreed. I previously thought so but in the example `auth0.rs` it needed `Algorithm` from `KeyAlgorithm` at `Validation::new`. I think we don't need public method just for that, we can do it via `Algorithm::from_str`